Dockerfile

```
@@ -1,10 +1,13 @@
-FROM eclipse-temurin:17-jre
-ENV TZ=Asia/Seoul
+FROM eclipse-temurin:17-jre-alpine
 ARG JAR_FILE=build/libs/*.jar
 
-RUN apt-get update && apt-get install -y curl && rm -rf /var/lib/apt/lists/*
+RUN apk add --no-cache curl tzdata && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)